include/fifo.h

```diff
@@ -49,10 +49,14 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
+/**
+ * @struct fifo_element
+ * @brief Structure holding the fifo element parameters.
+ */
 struct fifo_element {
-	struct fifo_element *next;
-	char *data;
-	uint32_t len;
+	struct fifo_element *next; ///< next FIFO element
```

Comment:
Why did you decide to place the documentation block after the member (shouldn't we do it before, the same as for the function's parameters for example)?